src/utils/axiosAPI/constants.js

```diff
@@ -0,0 +1,3 @@
+export const BASE_URL = 'https://api.nasa.gov/'
+export const GALLERY_URL = 'https://images-api.nasa.gov'
+export const API_KEY = 'api_key=aCNwLvVe8gykdUxPUggWkOt3lB3RzzHoVHTy4ZQZ'
```

Comment:
this should be in another file - in src/common/constants, because its not only an axios constant but a common one